### PR TITLE
add option to change qscout entangler

### PR DESCRIPTION
**Pull request by AndrewLitteken**

### CODE REVIEW

**Comment by richrines1** [comment]
same request as @dowusu-antwi's [in qss](https://github.com/SupertechLabs/qiskit-superstaq/pull/295#discussion_r1067275754) (otherwise lgtm)

**Comment by AndrewLitteken** [comment]
Updated with a check client side for changing the entangler.

**Comment by AndrewLitteken** [comment]
@richrines1 could you take a look to see if this is what you meant?

File: cirq_superstaq/service.py [hunk 1]
```diff
@@ -400,8 +401,10 @@ def qscout_compile(
         """
         serialized_circuits = css.serialization.serialize_circuits(circuits)
         circuits_is_list = not isinstance(circuits, cirq.Circuit)
+        if base_entangling_gate not in ("xx", "zz"):
+            raise ValueError("base_entangling_gate must be either 'xx' or 'zz'")
```
Comment (richrines1) [inline cirq_superstaq/service.py:405]: nit: check before bothering to serialize anything
```suggestion
        if base_entangling_gate not in ("xx", "zz"):
            raise ValueError("base_entangling_gate must be either 'xx' or 'zz'")

        serialized_circuits = css.serialization.serialize_circuits(circuits)
        circuits_is_list = not isinstance(circuits, cirq.Circuit)
```

**Comment by AndrewLitteken** [comment]
should be fixed now

**Comment by richrines1** [comment]
> should be fixed now

i think this update still needs to be pushed?

**Comment by AndrewLitteken** [comment]
Yes, but I think @pgoiporia needs to approve it.

**Comment by AndrewLitteken** [comment]
Ok it should actually be up now

**Review by richrines1 — APPROVED** [review]
nice, lgtm!

**Pull request merged**